### PR TITLE
Initialize visual flow

### PR DESCRIPTION
**Pull request by HimashiRathnayake**
## Purpose
This PR initializes the visual flow and adds the  svg components required in the visual flow of the Authentication Flow Composer.

## Goals

- Initialize the visual flow
- Add svg components required in Authentication Flow Composer.

## Approach
image

### CODE REVIEW

File: src/components/core/svg/authentication-option-button.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    props: AuthenticationOptionButtonProps
+) => {
+
+    const {
+        type,
+        x,
+        y
+    } = props;
+
+    const iconX = x+10;
```
Comment (brionmario) [inline src/components/core/svg/authentication-option-button.tsx]: Check other places as well.
```suggestion
    const iconX: number = x + 10;
```

File: src/components/core/svg/identifier-first.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+) : ReactElement => {
+    
+    const { 
+        options 
+    } = props;
+    
+    const Y= options.length*40;
+    const height = 780+Y;
+    const viewBox = `0 0 526 ${height}`;
+    return(
```
Comment (brionmario) [inline src/components/core/svg/identifier-first.tsx:51]:
```suggestion

    return(
```

File: src/components/core/svg/totp.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+import React, { FunctionComponent, ReactElement } from "react";
+
+/**
+ * Totp Authenticator SVG Component.
+ *
+ * @return {React.ReactElement}
+ */
+export const Totp: FunctionComponent = () : ReactElement => {
+    return(
```
Comment (brionmario) [inline src/components/core/svg/totp.tsx:28]:
```suggestion
   
    return(
```

**Review by brionmario — APPROVED** [review]

File: src/components/core/svg/authentication-option-button.tsx (hunk 1 above)
Comment (HimashiRathnayake) [reply src/components/core/svg/authentication-option-button.tsx]: Fixed with [01a90d6](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/7/commits/01a90d62848ee7620ed50c42371af7202f2532af)

**Pull request merged**